code-reviews/recipes/JavaScript.md

```diff
@@ -2,27 +2,136 @@
 
```

Comment:
I am wondering if we should also rename the file to reflect JS & TS

Reply:
Sure, I'll do this when I have gotten responses to all the other comments, and will do all changes at once.

Reply:
Fixed